ex.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from PIL import Image
+import base64
+import os
+from PIL import Image as im
+import datetime
+import pickle5 as pickle
+from datetime import datetime
+def ex1():
+    rootdir = r'C:\Users\omri tuito\Desktop\pythonProject8\ex1'
+    for file in os.listdir(rootdir):
+        old_file_name = os.path.join(rootdir, file)
+        decode_name = base64.b32decode(file).decode('utf-8')
+        new_file_name = os.path.join(rootdir, decode_name)
+        os.rename(old_file_name,new_file_name)
+
+def ex2():
+    img = Image.open(r'C:\Users\omri tuito\Desktop\pythonProject8\ex2\noised_img.png')
+    data_from_hacker = np.asarray(img)
+    data_from_mysterious = np.load(r'C:\Users\omri tuito\Desktop\pythonProject8\ex2\mysterious_file.npz')
+    data_from_mysterious = data_from_mysterious['arr_0']
+    data_from_mysterious = im.fromarray(data_from_hacker - data_from_mysterious)
+    data_from_mysterious.save('decode_image.png')
+
+
+def get_dict():
```

Comment:
You should make helper functions more generic by sending them variables, rather than using globals. It makes sense for your 'main' function but not for the others